symfony-app-skeleton/src/Manager/ShipmentOrderManager.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Manager;
+
+use App\AutoMapping;
+use App\Entity\OrderShipmentEntity;
+use App\Repository\OrderShipmentEntityRepository;
+use App\Request\OrderShipmentCreateRequest;
+use App\Request\ShipmentOrderStatusUpdateRequest;
+use App\Request\ShipmentStatusCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+
+class ShipmentOrderManager
+{
+    private $autoMapping;
+    private $entityManager;
+    private $orderShipmentEntityRepository;
+    private $shipmentStatusManager;
+
+    public function __construct(AutoMapping $autoMapping, EntityManagerInterface $entityManager, OrderShipmentEntityRepository $orderShipmentEntityRepository,
+     ShipmentStatusManager $shipmentStatusManager)
+    {
+        $this->autoMapping = $autoMapping;
+        $this->entityManager = $entityManager;
+        $this->orderShipmentEntityRepository = $orderShipmentEntityRepository;
+        $this->shipmentStatusManager = $shipmentStatusManager;
+    }
+
+    public function createShipmentOrder(OrderShipmentCreateRequest $request)
+    {
+        $orderShipmentEntity = $this->autoMapping->map(OrderShipmentCreateRequest::class, OrderShipmentEntity::class, $request);
+        
+        $this->entityManager->persist($orderShipmentEntity);
+        $this->entityManager->flush();
+        $this->entityManager->clear();
+
+        return $orderShipmentEntity;
+    }
+
+    public function getShipmentsOrdersByStatus($status)
+    {
+        return $this->orderShipmentEntityRepository->getShipmentsOrdersByStatus($status);
+    }
+
+    public function updateShipmentOrderStatus(ShipmentOrderStatusUpdateRequest $request)
+    {
+        // First, we update the status of the order
+
+        $shipmentOrderEntity = $this->orderShipmentEntityRepository->find($request->getId());
+        
+        if(!$shipmentOrderEntity)
+        {
+            return null;
+        }
+        else
+        {
+            $shipmentOrderEntity = $this->autoMapping->mapToObject(ShipmentOrderStatusUpdateRequest::class, OrderShipmentEntity::class, 
```

Comment:
If this just to update the status of the order shipment to be accepted, so we do not need the frontend to send us order details just _**orderID**_ will be enough you add status like this then flush:
`shipmentOrderEntity->setStatus('accepted')`

**Try not to depend on the front end to send keyword that represents the status**